scripts/jenkins/cloud/manual/deployer-entry-point.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash -i
+
+: ${git_user_name:=Unknown User}
+: ${git_user_email:=unknown@unknown.org}
+: ${python:=python2}
+
+configure_git() {
+    git config --global user.name "${git_user_name}"
+    git config --global user.email "${git_user_email}"
+}
+
+set_up_venv() {
+    ln -s "/opt/ansible-venv-${python}" /opt/ansible-venv
```

Comment:
the ANSIBLE_STRATEGY_PLUGINS env needs to be configured for mitogen to work correctly. 
it takes the form: 
export ANSIBLE_STRATEGY_PLUGINS=/opt/ansible-venv-python2/lib/python2.7/site-packages/ansible_mitogen/plugins/strategy

but it obviously depends on the version of pytyhon